ups/ap_pipe.cfg

```diff
@@ -0,0 +1,26 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+# Dependencies that provide header files and or libraries should be included here.
+# Pure-Python dependencies do not need to be included.
+# Packages that use pybind11 or boost_tests should declare them as build dependencies.
+# Otherwise, the rules for which packages to list here are the same as those for
+# table files.
+dependencies = {
+    "required": ["utils", "ndarray"],
+    "buildRequired": ["boost_test", "pybind11", "ndarray", "pex_exceptions", "obs_decam"],
+    "optional": ["ap_verify_hits2015", "ap_verify"],
```

Comment:
I saw yours was empty for `dependencies`; I went with the one in the template repo and just added `obs_decam`. But I suspect you're right about not needing to have anything in here for python-only dependencies, so I'll make it empty instead.